mlflow/server/js/src/experiment-tracking/components/ExperimentPage.tsx

```diff
@@ -17,7 +17,7 @@ const ExperimentPage = () => {
   const { theme } = useDesignSystemTheme();
 
   const { tabName } = useParams();
-  const shouldRenderTabbedView = isExperimentLoggedModelsUIEnabled() && Boolean(tabName);
+  const shouldRenderTabbedView = shouldEnableExperimentPageChildRoutes() || Boolean(tabName);
```

Comment:
after recent changes, we have both `experiment-tracking/components/experiment-page/ExperimentPage.tsx` and `experiment-tracking/components/ExperimentPage.tsx` now, it would be good to consolidate it now